viewing_party/party.py

```diff
@@ -1,23 +1,163 @@
 # ------------- WAVE 1 --------------------
+from statistics import mean, mode
 
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
+    if title and genre and rating:
+    
+        movie = {"title": title,
+    "genre": genre,
+    "rating": rating,
+    }
+
+        return movie
+ 
+def add_to_watched(user_data, movie):
+
+    movies_watched = movie
+    user_data["watched"].append(movies_watched)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+
+    movie_watchlist = movie

```

Comment:
Same here, We are we reassigning the variable here? We can simply append the movie that's being passed in.